airbyte-scheduler/src/main/resources/image_exists.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -e
+
+function _error() {
+  echo "$@" && exit 1
+}
+
+function _usage() {
+  _error "Usage: ./image_exists.sh imageName tag"
+}
+
+function docker_exists_in_remote() {
+  curl --silent -f -lSL https://index.docker.io/v1/repositories/$1/tags/$2 > /dev/null
```

Comment:
that's incorrect as images can be hosted elsewhere. Why not just a docker pull ?

Reply:
i was trying to avoid docker pull since it can be expensive to run. trying to avoid has been a hassle though, you're right, i'll just use docker pull.